teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>I suggest the following hierarchy (so that PCM is not the only bullet point on level 1) - this will look better later on our webpage where we have the TOC on right (see example T001 https://projects.volkamerlab.org/teachopencadd/talktorials/T001_query_chembl.html).</p><p> </p><ul><li>Proteochemometrics (PCM) modelling</li><li>Data preparation</li><li class="ql-indent-1">Papyrus dataset</li><li class="ql-indent-1">Molecule encoding: molecular descriptors</li><li class="ql-indent-1">Protein encoding: protein descriptors</li><li>Machine learning principles: <strong>R</strong>egression</li><li class="ql-indent-1">Data splitting methods</li><li class="ql-indent-1">Regression evaluation metrics</li><li class="ql-indent-1">ML algorithm: Random Forest</li><li>Applications of PCM in drug discovery</li></ul><p> </p><p>This new hierarchy would also need updating in the actual header in the Theory section (##, ###, etc.).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I am not sure if this is just me, but I see all points with the same hierarchy in your comment. </p><p>Maybe:</p><ul><li>Proteochemometrics (PCM) modelling</li><li>Data preparation</li><li class="ql-indent-1">Papyrus dataset</li><li class="ql-indent-1">Molecule encoding: molecular descriptors</li><li class="ql-indent-1">Protein encoding: protein descriptors</li><li>Machine learning principles:&nbsp;<strong>R</strong>egression</li><li class="ql-indent-1">Data splitting methods</li><li class="ql-indent-1">Regression evaluation metrics</li><li class="ql-indent-1">ML algorithm: Random Forest</li><li>Applications of PCM in drug discovery</li></ul><p> </p>

Reply:
@gorostiolam, sorry for the formatting issue - your hierarchy is exactly what I meant, thank you!